### DISCUSSION TITLE
KR norm on raw.cool file

### DISCUSSION THREAD

**Discussion by matteozoia4** [open2c/cooler#272] [Q&A] [answered]
Dear all,

which one would be the standard procedure to assess KR norm on raw.cool file?

Use the "--annotate KR" option or select a specific option with  "cooler balance ..."

Many thanks for the help!

M-Z

**Comment by Phlya** [comment]
Could you clarify what you mean?

Cooler doesn't have KR balancing implemented, but the result of IC procedure it does is the same - except for more stringent filtering of "bad" bins before correction is applied.

**Comment by matteozoia4** [comment]
Hi,



Many thanks for your quick answer.


Sorry I am indeed a beginner in Cooler tool usage, and reading through the readthedocs I was unsure of the procedure to apply.


Which would then be the command and options to employ in case I would like to pursue with the IC procedure (KR with more stringent filtering) you mentioned ?


Many thanks for your time and help, very appreciated!



M.Z




Zoia Matteo, Ph.D. Student
Gene Regulation & Cardiac Development

Marco Osterwalder Laboratory
Department for BioMedical Research (DBMR)
University of Bern
Lab 369, Murtenstrasse 28
CH - 3008 Bern
SWITZERLAND

________________________________
Da: Ilya Flyamer ***@***.***>
Inviato: domenica, 13. marzo 2022 10:25:24
A: open2c/cooler
Cc: Zoia, Matteo (DBMR); Author
Oggetto: Re: [open2c/cooler] KR norm on raw.cool file (Discussion #272)


Could you clarify what you mean?

Cooler doesn't have KR balancing implemented, but the result of IC procedure it does is the same - except for more stringent filtering of "bad" bins before correction is applied.

—
Reply to this email directly, view it on GitHub<https://github.com/open2c/cooler/discussions/272#discussioncomment-2349269>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AUBHT3EFGTWDIK524T7IEW3U7WYAJANCNFSM5QTB44JQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Comment by Phlya** [comment, marked as answer]
You just use `cooler balance <your_file.cool>`. Check the additional arguments, in case there is something useful (multiple cores, for example).
In case you want to create a multiresolution cooler by coarsegraining a high resolution file, you can just use `cooler zoomify` with --balance argument, then very resolution will be balanced too.

**Comment by matteozoia4** [comment]
Hi,


Many thanks for the help and the insights!

Both steps are indeed useful!


MZ


Zoia Matteo, Ph.D. Student
Gene Regulation & Cardiac Development

Marco Osterwalder Laboratory
Department for BioMedical Research (DBMR)
University of Bern
Lab 369, Murtenstrasse 28
CH - 3008 Bern
SWITZERLAND

________________________________
Da: Ilya Flyamer ***@***.***>
Inviato: domenica, 13. marzo 2022 11:06:50
A: open2c/cooler
Cc: Zoia, Matteo (DBMR); Author
Oggetto: Re: [open2c/cooler] KR norm on raw.cool file (Discussion #272)


You just use cooler balance <your_file.cool>. Check the additional arguments, in case there is something useful (multiple cores, for example).
In case you want to create a multiresolution cooler by coarsegraining a high resolution file, you can just use cooler zoomify with --balance argument, then very resolution will be balanced too.

—
Reply to this email directly, view it on GitHub<https://github.com/open2c/cooler/discussions/272#discussioncomment-2349380>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AUBHT3FL7MV233DYU5ODXDDU7W43VANCNFSM5QTB44JQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
